spaceship/lib/spaceship/portal/certificate.rb

```diff
@@ -258,7 +267,7 @@ def factory(attrs)
         #  only include certificates matching the current type.
         def all(mac: false)
           if self == Certificate # are we the base-class?
-            type_ids = mac ? MAC_CERTIFICATE_TYPE_IDS : IOS_CERTIFICATE_TYPE_IDS
+            type_ids = mac ? MAC_CERTIFICATE_TYPE_IDS : APPLE_CERTIFICATE_TYPE_IDS.merge(IOS_CERTIFICATE_TYPE_IDS)
```

Comment:
Probably merge `APPLE_CERTIFICATE_TYPE_IDS` for both?

Reply:
I just added a commit that fixed this ^

Reply:
Thank you, I was wondering what is expected behavior in case `mac` is `true` here.
Probably this is expected, but prefer to defer to the decision to someone who knows expected behavior here.